### DISCUSSION TITLE
VIEW in HASURA

### DISCUSSION THREAD

**Discussion by IzaCaroline** [hasura/graphql-engine#9361] [Question] [answered]
accordingly:
[Views](https://www.postgresql.org/docs/current/sql-createview.html) 
can be used to expose the results of a custom query as a virtual table. Views are not physically persisted, that is, the query that defines a view is executed whenever data is requested from the view.

that is: 
the views created it updates itself according to the data update or should I update it alone?

![image](https://user-images.githubusercontent.com/99855235/210874486-27937889-935a-43c6-95d3-cfafc64e5af3.png)

**Comment by m-rgba** [comment, marked as answer]
Views should update themselves following a change based on introspecting your database.
If you find that a view hasn't updated on Hasura's side (for instance if you make a change using an external SQL editor), you can force a reload of the database by going to `Data > Manage` and selecting `Reload` on your your database's row.

![image](https://user-images.githubusercontent.com/74692114/210875651-448a6190-5d0c-44e8-abfb-bde7edfefb01.png)

**Reply by IzaCaroline** [reply]
Thank you very much :)
